src/Simplex_tree/include/gudhi/Simplex_tree/Simplex_tree_node_explicit_storage.h

```diff
@@ -43,7 +43,13 @@ struct GUDHI_EMPTY_BASE_CLASS_OPTIMIZATION Simplex_tree_node_explicit_storage
   typedef typename SimplexTree::Simplex_key Simplex_key;
   typedef typename SimplexTree::Simplex_data Simplex_data;
 
-  Simplex_tree_node_explicit_storage(Siblings* sib = nullptr, Filtration_value filtration = 0)
+  // Simplex_tree_node_explicit_storage() : children_(nullptr) {}
```

Comment:
Because it was in conflict with the other constructor. But I just have to remove the default value there and than it should work. Those constructors changed a lot back and forth...